packages/project-disaster-trail/src/components/Game/Orb.js

```diff
@@ -92,41 +88,44 @@ class Orb extends Component {
   };
 
   render() {
-    const { animationState, orbSize } = this.state;
+    const { animationState } = this.state;
+    const { size } = this.props;
 
-    const circleSizeStyle = css`
-      height: ${orbSize}px;
-      width: ${orbSize}px;
-      border-radius: ${orbSize}px;
+    const sizeStyle = css`
+      height: ${size}px;
+      width: ${size}px;
+      border-radius: ${size}px;
     `;
 
-    const orbSizeStyle = css`
-      height: ${orbSize + 20}px;
-      width: ${orbSize + 20}px;
-      border-radius: ${orbSize + 20}px;
+    const absoluteStyle = css`
+      position: absolute;
+      top: -${size / 2}px;
+      left: -${size / 2}px;
```

Comment:
Nice. I was so confused why things were in weird places

Reply:
Silly css layouts are always a pain!